Maestro/subscriptions.json

```diff
@@ -299,6 +305,13 @@
         }
       }
     },
+    // Mirror corefx changes into VSO
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/**/*",
+      ],
+      "action": "corefx-mirror"
+    },
```

Comment:
Added targets for the other branches that actually exist (I didn't add dev/api or dev/eng, since those branches don't exist in corefx anymore, so there is no reason to add triggers that will never fire)